eng/Versions.props

```diff
@@ -128,8 +128,7 @@
     <VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>3.0.0-beta2-18612-05</VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>
     <VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>3.0.0-beta2-18612-05</VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
-    <XunitPackageVersion>2.3.1</XunitPackageVersion>
-    <XunitRunnerVisualStudioPackageVersion>2.4.0</XunitRunnerVisualStudioPackageVersion>
+    <XunitVersion>2.4.1</XunitVersion>
```

Comment:
Odd. aspnet/Extensions is on `2.4.0` and that bumping things up here would cause different package downgrade errors. Glad it doesn't.

Reply:
> Should we move aspnet/Extensions to Xunit `2.4.1` and (then) the other repos?

Yes, but I would do it in reverse order. Move AspNetCore to 2.4.1, then Extensions

Reply:
> Move AspNetCore to 2.4.1, then Extensions

In `release/3.0-preview3` or `master`?

Reply:
master. It's not urgent for preview3